tests/integration/suite/daprd/scheduler/streaming/stream.go

```diff
@@ -96,6 +97,7 @@ func (s *streaming) Setup(t *testing.T) []framework.Option {
 		daprd.WithExecOptions(
 			exec.WithStdout(s.streamloglineDaprB.Stdout()),
 		),
+		daprd.WithLogLevel("debug"),
```

Comment:
I thought this test was still valuable, as it ensures that the sidecar receives the job